crates/cheatcodes/src/inspector.rs

```diff
@@ -77,6 +78,215 @@ impl Context {
     }
 }
 
+/// Common behaviour of legacy and EOF create inputs.
+trait CommonCreateInput<DB: DatabaseExt> {
```

Comment:
I get why we need this, imo this an appropriate solution.

the inspector.rs is already gigantic. could we perhaps convert this into a module and put this internal helpers into `inspector/utils.rs` or common.rs

Reply:
makes sense, done here https://github.com/foundry-rs/foundry/commit/baac32e22fbd5ab50d40ec4a8c16009b34ebe2cd